### ISSUE TITLE
Song Not Playing?

### ISSUE THREAD

**Issue by yusufricky** [Asmaa-k/Audio-Stream#1]
Hallo bro, why this it song if i want select list (than notif change in home device) but not  playing? seek not reading and voice song not playing? big thanks, please reply :)

**Comment by Asmaa-k** [comment]
I think the problem is the songs on firebase are not available anymore
because I upload them to a free host from months ago , I think the free hostess deleted my site after a while with songs file

**Comment by yusufricky** [comment]
Yes i agree, another problem bro, i replace app>google-service.json with my account console firebase and create firestore but why error result?

method 'java.lang.Object java.util.Map.get(java.lang.Object)' on a null object reference
        at com.codingwithmitch.audiostreamer.ui.HomeFragment$1.onComplete(HomeFragment.java:86)
